x/did/tests/setup/utils.go

```diff
@@ -26,16 +28,30 @@ func randBase58Seq(bytes int) string {
 type IDType int
 
 const (
-	Base58_16bytes IDType = iota
-	UUID           IDType = iota
+	Base58_16bytes   IDType = iota
+	Base58_16symbols IDType = iota
+	UUID             IDType = iota
 )
 
+var letters = []rune("123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz")
```

Comment:
Why do we need this now when we have a Base58 library? Also, better to generate this using Indy algorithm.